src/main.jsx

```diff
@@ -1,6 +1,6 @@
 import "./theme/defaults.scss";
 import "bootstrap/dist/css/bootstrap.min.css";
-import React from "react";
+import React, { useState } from "react";
```

Comment:
I don't think you need this import here

Reply:
In fact, could you install this package to allow prettier to automatically organize and remove unused imports?

https://www.npmjs.com/package/prettier-plugin-organize-imports

It should just be a matter of running the install command,  you shouldn't need to update any prettier configurations. You can verify that it's working by checking that it fixes this line